runtime/doc/autocmd.txt

```diff
@@ -1230,6 +1233,8 @@ VimLeavePre			Before exiting Vim, just before writing the
 VimResized			After the Vim window was resized, thus 'lines'
 				and/or 'columns' changed.  Not when starting
 				up though.
+							*VimResumed*
```

Comment:
I think it should be `*VimResume*` and not `*VimResumed*` (note the extra final "d"). Same thing on the next line.